crates/sui-graphql-rpc/src/config.rs

```diff
@@ -16,6 +16,10 @@ const MAX_QUERY_PAYLOAD_SIZE: u32 = 2_000;
 const MAX_DB_QUERY_COST: u64 = 20_000; // Max DB query cost (normally f64) truncated
 const DEFAULT_PAGE_SIZE: u64 = 20; // Default number of elements allowed on a page of a connection
 const MAX_PAGE_SIZE: u64 = 50; // Maximum number of elements allowed on a page of a connection
+const MAX_TYPE_ARGUMENT_DEPTH: u32 = 16;
+const MAX_TYPE_ARGUMENT_WIDTH: u32 = 32;
```

Comment:
Comment: was wondering where we got these default values from, looks like all but the max_type_argument_width have corresponding default values in sui-protocol-config. I feel like a generic type with 32 params would be crazy, so seems sensible enough to me lol

Reply:
`max_type_argument_width` is also from our protocol configs or VM limits, under a different name:

https://github.com/MystenLabs/sui/blob/6db17d1d3b4565d8b91b3419d470adffda137eb7/crates/sui-protocol-config/src/lib.rs#L1192

Reply:
ah i see